src/hooks/useStore.ts

```diff
@@ -14,7 +14,7 @@ export function createStoreHook(context = ReactReduxContext) {
       ? useDefaultReduxContext
       : () => useContext(context)
   return function useStore() {
-    const { store } = useReduxContext()
+    const { store } = useReduxContext()!
```

Comment:
Since the context is of type `ReactReduxContextValue | null`, we can't just destructure this. To keep the existing API intact we can override the compiler here and preserve existing behaviour.

Keen to hear your thoughts on whether this can/should be changed though.

Reply:
Fine by me. It's _theoretically_ possible_ for this to fail, but in practice it shouldn't, and it _should_ be a runtime error if there's no store. Making the assumption that it exists is the best option here.